babel/messages/catalog.py

```diff
@@ -142,6 +142,13 @@ def __eq__(self, other):
     def __ne__(self, other):
         return self.__cmp__(other) != 0
 
+    def is_identical(self, other):
+        """Checks whether messages are identical, taking into account all
+        properties.
+        """
+        assert isinstance(other, Message)
+        return self.__dict__ == self.__dict__
```

Comment:
```suggestion
        return self.__dict__ == other.__dict__
```
😅

Reply:
Haha oopsie... Thanks for being thorough! I have added a regression test for this.